Code/Engine/RendererFoundation/Shader/Types.h

```diff
@@ -30,6 +30,31 @@ class ezShaderMat3
   float m_Data[12];
 };
 
+/// \brief A wrapper class that converts a ezMat4 into the correct data layout for shaders.
```

Comment:
We have used ezMat4 directly for constant buffers etc. so far, why is this needed?

Reply:
Because of bugs in other APIs: https://issues.chromium.org/issues/364267168
Basically it allows me to inject a matrix transpose at some point.